frontend/DataView/DataView.js

```diff
@@ -74,9 +74,20 @@ class DataView extends React.Component<DataViewProps> {
       return <div style={missingStyle(theme)}>null</div>;
     }
 
+    const dataType = typeof data;
+    var isPrimitive = dataType === 'number' || dataType === 'string' || dataType === 'boolean';
```

Comment:
Found that as well. I'm pretty sure it's unrelated to the changes because numbers are not displayed when testing on the master branch. I'll probably open an issue for that.